lib/core/command.dart

```diff
@@ -140,6 +140,17 @@ class Commands {
         return self;
       },
     ),
+    Command(
+      prefix: '\@enum',
+      command: '',
```

Comment:
looks like the code is not same as in the README.md (i didnt tested yet)
```suggestion
      command: ':',
```